modules/customize-certificates-add-service-serving-validating-webhook.adoc

```diff
@@ -14,27 +14,27 @@ Do not set this annotation for admission webhook configurations that need to spe
 
 .Procedure
 
-. Annotate the ValidatingWebhookConfiguration with `service.beta.openshift.io/inject-cabundle=true`:
+. Annotate the validating webhook configuration with `service.beta.openshift.io/inject-cabundle=true`:
 +
 [source,terminal]
 ----
-$ oc annotate validatingwebhookconfigurations <validatingwebhook-name> \//<1>
+$ oc annotate validatingwebhookconfigurations <validating_webhook_name> \//<1>
      service.beta.openshift.io/inject-cabundle=true
 ----
-<1> Replace `<validatingwebhook-name>` with the name of the ValidatingWebhookConfiguration to annotate.
+<1> Replace `<validating_webhook_name>` with the name of the validating webhook configuration to annotate.
 +
-For example, use the following command to annotate the ValidatingWebhookConfiguration `foo`:
+For example, use the following command to annotate the validating webhook configuration `foo`:
```

Comment:
In OLM @adellape was excising instances of _foo_ with _exampleN_. Do we want to keep _foo_?

Reply:
Sure, I don't mind updating to remove references to "foo" here.